CONTRIBUTING.md

```diff
@@ -0,0 +1,200 @@
+   - `npm run build` - Build for production
+   - `npm run preview` - Preview production build
+
+## Types of Contributions
+
+We welcome various types of contributions:
+
+- **Blog posts** - Share news, tutorials, or insights about Spyder
+- **Bug fixes** - Fix issues in the website functionality
+- **Feature enhancements** - Improve existing features or add new ones
+- **Documentation** - Improve clarity and completeness of documentation
+
+## Creating Blog Posts
+
+### Adding New Authors
+
+If the blog post author(s) haven't been added yet:
+
+1. **Create author folder** in `static/assets/authors/` using a simple identifier (no spaces/special characters)
+2. **Add author files**:
+   - Square avatar image (recommended: 512×512px WEBP format)
+   - `metadata.json` file:
+   ```json
+   {
+       "name": "Author's Display Name",
+       "image": "filename.webp"
+   }
+   ```
+
+### Creating the Blog Post
+
+1. **Create post folder** in `src/routes/blog/` (folder name becomes URL slug - use lowercase, no spaces)
+
+2. **Add main content file** `+page.md` with proper frontmatter:
+   ```yaml
+   ---
+   title: Blog Post Title # Required
+   author: author-id  # Required. For multiple authors, use array: [author1, author2]
+   pub_date: YYYY-MM-DD # Required (ISO format)
+   category: Category Name  # Optional
+   tags: Tag1, Tag2  # Optional (comma-separated)
+   summary: Concise overview of content # Required
+   ---
+   ```
+
+3. **Write content** using mdsvex Markdown:
+   - [Never indent code blocks](https://mdsvex.pngwn.io/docs#limitations)
+   - Use standard Markdown syntax for formatting
+   - Follow our [style guide](#style-guidelines) for consistency
+
+4. **Add images** (if needed):
+   ```markdown
+   ![Meaningful description](image.webp "Optional caption")
+   ```
+   - Store images in the post's folder
+   - Use descriptive alt text (avoid "image of...", "picture of...", etc.)
+   - Prefer WEBP format for better performance
+
+5. **Preview your post**:
+   ```bash
+   npm run dev
+   ```
+   - Verify post appears at `/blog/your-post-name`
+   - Check author metadata and image display
+   - Test on different screen sizes
+
+## Code Contributions
+
+### Guidelines
+
+- Follow the existing code style and conventions
+- Write clear, descriptive commit messages
+- Test your changes thoroughly
+- Update documentation if needed
+
+### File Structure
+
+- `src/routes/` - Page routes and content
+- `src/lib/` - Reusable components and utilities
+- `static/assets/` - Static assets (images, fonts, etc.)
+- `src/app.css` - Global styles
+
+## Reporting Issues
+
+Before creating an issue:
+
+1. **Search existing issues** to avoid duplicates
+2. **Use the issue template** (if available)
+3. **Include relevant details**:
+   - Clear description of the problem
+   - Steps to reproduce
+   - Expected vs. actual behavior
+   - Browser/OS information (if relevant)
+   - Screenshots (if applicable)
+
+## Pull Request Process
```

Comment:
The first step can be adapted to a section to the template-integrated version as described above (between "Prepare" and "Commit"), while the rest are already taken care of there.